packages/common/src/utils/filters.ts

```diff
@@ -425,6 +425,20 @@ const flattenSameFilterGroupType = (filterGroup: FilterGroup): FilterGroup => {
     } as FilterGroup;
 };
 
+/**
+ * Checks if a dimension value is an invalid date before it is added to the filter
+ * @param item - The field to compare against the value
+ * @param value - The value to check
+ * @returns True if the value is an invalid date, false otherwise
+ */
+export const isDimensionValueInvalidDate = (
+    item: FilterableField,
+    value: any,
+) =>
+    isDimension(item) &&
+    Object.values(TimeFrames).includes(item.timeInterval as TimeFrames) &&
```

Comment:
Is this just checking its a date type? Could you use isDateItem?

Reply:
Yes... my intellisense just wasn't working 😥 thank you!